heron/common/src/java/com/twitter/heron/common/utils/misc/SerializeDeSerializeHelper.java

```diff
@@ -33,7 +36,12 @@ public static IPluggableSerializer getSerializer(Map<String, Object> config) {
     try {
       String serializerClassName = (String) config.get(Config.TOPOLOGY_SERIALIZER_CLASSNAME);
       if (serializerClassName == null) {
-        serializer = new KryoSerializer();
+        LOG.log(Level.WARNING, "Serializer class name not provided");
+        LOG.log(Level.WARNING, "Fall back to Java serializer. "
+                + "This could cause serious performance degradation");
+        LOG.log(Level.WARNING, "You can specify to use Kryo as serializer. "
```

Comment:
Can you please include these all in one log message? It's typically better to have the entire log entry in one longer warning without line breaks, than to have it split into multiple lines.